website/project/views/node.py

```diff
@@ -38,7 +38,7 @@
 from website.profile import utils
 from website.project import new_folder
 from website.util.sanitize import strip_html
-
+from website.spam_admin.utils import _project_is_spam
```

Comment:
When I first created it, this function was inside Node.py, thus it made sense to make it hidden. However, I eventually moved it to spam_admin utils to in order to centralize all my code. It is still not being used anywhere else so I thought making it anonymous is okay. However, I do see your point. I am importing it so its not technically hidden anymore. I will change it.